Components/AccordionVertical/script.js

```diff
@@ -0,0 +1,130 @@
+import $ from 'jquery'
+
+class AccordionVertical extends window.HTMLDivElement {
+  constructor (self) {
+    self = super(self)
+    self.$ = $(self)
+    self.setOptions()
+    self.resolveElements()
+    return self
+  }
+
+  setOptions () {
+    this.aria = {
+      hidden: 'aria-hidden',
+      controls: 'aria-controls',
+      expanded: 'aria-expanded'
+    }
+    this.class = {
+      expanded: 'accordionVertical-item-isExpanded',
+      hidden: 'accordionVertical-item-isHidden',
+      contentIsSizing: 'accordionVertical-content-isSizing'
+    }
+    this.isMultiselectable = this.$.attr('aria-multiselectable')
+  }
+
+  resolveElements () {
+    this.$item = $('.accordionVertical-item', this)
+    this.$button = $('.accordionVertical-btn[aria-controls]', this)
+    this.$content = $('.accordionVertical-content', this)
+  }
+
+  connectedCallback () {
+    this.$.on('click', this.$button.selector, this.toggleButton)
+    $(window).on('load resize', this.updateExpandedItemHeight)
+  }
+
+  /**
+   * Toggle item state for clicked button.
+   */
+  toggleButton = (e, expanded) => {
+    e.preventDefault()
+    const $button = $(e.currentTarget)
+    this.toggleItem($button, expanded)
+  }
+
+  /**
+   * Toggle an item's "expanded" state, optionally providing a target
+   * state.
+   *
+   * @param {HTMLButtonElement} button
+   * @param {boolean?} expanded If no state is provided, the current
+   * state will be toggled (from false to true, and vice-versa).
+   * @return {boolean} the resulting state.
+   */
+  toggleItem ($button, expanded) {
+    const id = $button.attr(this.aria.controls)
+    const $item = $button.closest(this.$item.selector)
+    const $content = $(`#${id}`, this)
+
+    if (!$content.length) {
+      throw new Error(
```

Comment:
I figured it was better this way because this is the incorrect way to use the component and the developer should test and see this error when testing if they break it. Then there's no chance of the component failing silently. Do we have a consistent approach to error handling in the base components that I should follow instead? @bdbch @usingthesystem